openhtf/__init__.py

```diff
@@ -428,6 +428,14 @@ def wrap_or_copy(cls, func, **options):
     retval.options.update(**options)
     return retval
 
+  def _asdict(self):
```

Comment:
Ah, I didn't realize that name and doc weren't direct attributes. I could see `func` and `cls` getting removed because they're not serializable in `convert_to_base_types`, so that would make this function's behavior more obvious:

```py
def _asdict(self):
  asdict = data.convert_to_base_types(self)
  asdict.update(name=self.name, doc=self.doc)
  return asdict
```